<launch>
    <node pkg="controls" type="policy_control.py" name="policy_controller" respawn="false" output="screen">
        <param name="control_interval" value="0.01" />
        <param name="model_checkpoint_path" value="$(find controls)/policy_weights/standing" />
    </node>
</launch>